rosidl_cli/rosidl_cli/command/generate/__init__.py

```diff
@@ -38,6 +38,10 @@ def add_arguments(self, parser):
             '-ts', '--type-support', metavar='TYPESUPPORT',
             dest='typesupports', action='append', default=[],
             help='Target type supports for generation.')
+        parser.add_argument(
+            '-td', '--type-description', metavar='TYPE_DESCRIPTION',
```

Comment:
@frneer considering these are files, 
```suggestion
            '-td', '--type-description-file', metavar='PATH',
```
and so on and so forth would be more accurate.

Reply:
Done.